src/analysis/normed_space/lp_space.lean

```diff
@@ -629,6 +630,50 @@ end
 
 end normed_space
 
+section normed_star_group
+
+variables [Π (i : α), star_add_monoid (E i)] [Π i, normed_star_group (E i)]
+
+lemma _root_.mem_ℓp.star_mem {f : Π i, E i}
```

Comment:
Can you also add
```lean
@[simp] lemma _root_.mem_ℓp.star_iff {f : Π i, E i} : mem_ℓp (star f) p ↔ mem_ℓp f p :=
```
which should be provable in the same style as `neg_iff` above.

Reply:
Certainly.